_includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md

```diff
@@ -0,0 +1,102 @@
+## ActiveSpeakerObserver
+{: #ActiveSpeakerObserver}
+
+<section markdown="1">
+
+> `@inherits` [RtpObserver](#RtpObserver)
+
+An active speaker observer monitors the speech activity of the selected audio producers. It just handles audio producers (if [addProducer()](#rtpObserver-addProducer) is called with a video producer it will fail).
+
+Audio levels used for speech detection are read from an RTP header extension. No decoding of audio data is done. See [RFC6464](https://tools.ietf.org/html/rfc6464) for more information.
+
+</section>
+
+
+### Dictionaries
+{: #ActiveSpeakerDetection-dictionaries}
+
+<section markdown="1">
+
+#### ActiveSpeakerObserverOptions
+{: #ActiveSpeakerObserverOptions .code}
+
+<div markdown="1" class="table-wrapper L3">
+
+Field          | Type    | Description  | Required | Default 
+-------------- | ------- | ------------ | -------- | ----------
+`interval`     | Number  | Interval in ms for checking audio volumes. | No | 300
+`appData`      | Object  | Custom application data. | No | `{ }`
+
+</div>
+
+
+#### ActiveSpeakerObserverActivity
+{: #ActiveSpeakerObserverActivity .code}
+
+<div markdown="1" class="table-wrapper L3">
+
+Field          | Type    | Description  | Required | Default 
+-------------- | ------- | ------------ | -------- | ----------
+`producer`     | [Producer](#Producer) | The dominant audio producer instance. | Yes |
+
+</div>
+
+</section>
+
+
+### Properties
+{: #ActiveSpeakerObserver-properties}
+
+<section markdown="1">
+
+See also [RtpObserver Properties](#RtpObserver-properties).
+
+</section>
+
+
+### Methods
+{: #ActiveSpeakerObserver-methods}
+
+<section markdown="1">
+
+See also [RtpObserver Methods](#RtpObserver-methods).
+
+</section>
+
+
+### Events
+{: #ActiveSpeakerObserver-events}
+
+<section markdown="1">
+
+See also [RtpObserver Events](#RtpObserver-events).
+
+#### ActiveSpeakerObserver.on("dominantspeaker", fn(event))
+{: #ActiveSpeakerObserver-on-dominantspeaker .code}
+
+Emitted when a new dominant speaker is detected.
+
+<div markdown="1" class="table-wrapper L3">
+
+Argument  | Type    | Description   
+--------- | ------- | ----------------
+`dominantspeaker` | property&lt;[Producer](#Producer)&gt; | Producer Object
```

Comment:
What's that "property"?

```suggestion
`dominantSpeaker` | [Producer](#Producer) | Audio producer with most dominant audio level in the last interval
```

Reply:
I will make this change. The event is not fired for every interval. Perhaps I should make that clear. The interval setting for the class is only used for how often it is tested. An event will only be emitted when there is a dominant speaker change.

Reply:
btw. I am not sure where the property came from. Initially I was editing the file with a markdown editor, then switch to doing it with a regular text editor.